test/specs/commands/create_account.js

```diff
@@ -20,30 +20,96 @@ import * as then from '../../steps/3_then';
 
 describe('create account command', () => {
 	beforeEach(setUpCommandCreateAccount);
+	Given(
+		'a passphrase "minute omit local rare sword knee banner pair rib museum shadow juice" with private key "314852d7afb0d4c283692fef8a2cb40e30c7a5df2ed79994178c10ac168d6d977ef45cd525e95b7a86244bbd4eb4550914ad06301013958f4dd64d32ef7bc588" and public key "7ef45cd525e95b7a86244bbd4eb4550914ad06301013958f4dd64d32ef7bc588" and address "2167422481642255385L"',
+		given.aPassphraseWithPrivateKeyAndPublicKeyAndAddress,
+		() => {
+			Given(
+				'the passphrase is generated by the createMnemonicPassphrase function',
+				given.thePassphraseIsGeneratedByTheCreateMnemonicPassphraseFunction,
+				() => {
+					Given('an action "create account"', given.anAction, () => {
+						When('the action is called', when.theActionIsCalled, () => {
+							Then(
```

Comment:
This test could be nested more efficiently if `Given a valid integer corresponding to number of accounts "1"` was more deeply nested.